Contents/mods/Hydrocraft/media/scripts/Chemistry.txt

```diff
@@ -57,6 +62,7 @@ item HCGunPowder
         Type            = Normal,
         DisplayName     = Sulphur Powder,
         icon            = HCSulphurpowder,
+		DisplayCategory	= CraftMine,
```

Comment:
craftammo i think. Technically also used for vulcanized rubber, but I think tis more likely people will stack this with their stuff for making ammo